src/third_party/starboard/wpe/shared/player/player_internal.cc

```diff
@@ -457,6 +457,12 @@ enum class MediaType {
 
 constexpr char kClearSamplesKey[] = "fake-key-magic";
 
+#if SB_HAS(NATIVE_AUDIO)
+static constexpr GstStreamVolumeFormat volumeFormat = GST_STREAM_VOLUME_FORMAT_LINEAR;
+#else
+static constexpr GstStreamVolumeFormat volumeFormat = GST_STREAM_VOLUME_FORMAT_CUBIC;
+#endif
+
```

Comment:
I do not think making it dependent on NATIVE_AUDIO is the way to go since this effectively changes volume format for  platforms which does not define that (which is all currently AFAICS). This might be ok but then let's do it explicit and switch to CUBIC format for all but then passing 0-1 values as cubic volume does not look ok to me.